types/src/front/api_handlers/public/assistant.ts

```diff
@@ -37,8 +37,13 @@ export const PublicPostContentFragmentRequestBodySchema = t.type({
   content: t.string,
   url: t.union([t.string, t.null]),
   contentType: t.union([
-    t.literal("slack_thread_content"),
+    t.literal("text/plain"),
```

Comment:
You can infer this from the `supportedContentFragment`.

Reply:
See here: https://github.com/dust-tt/dust/blob/0d120635213c1436145ce4e6ad671bd237d32303/front/lib/api/pagination.ts#L21-L42.